docs/web-servers/squid/squid-http-proxy-centos-8/index.md

```diff
@@ -0,0 +1,234 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: This guide shows how to use Squid to create an HTTP proxy server on your Linode running Centos 8.
+og_description: This guide shows how to use Squid to create an HTTP proxy server on your Linode running Centos 8.
+keywords: ["squid", "proxy", "centos", "8", "http"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-04-14
+modified: 2020-04-14
+modified_by:
+  name: Rajakavitha Kodhandapani
+title: 'How to Create an HTTP Proxy Using Squid on CentOS 8'
+h1_title: 'Creating an HTTP Proxy Using Squid on CentOS 8'
+external_resources:
+ - '[Squid Official Site](http://www.squid-cache.org/)'
+ - '[Configure Proxy on Windows](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet)'
+ - '[Proxy Server Settings on macOS](https://support.apple.com/en-in/guide/mac-help/mchlp2591/mac)'
+ - '[Connection Settings in Firefox](https://support.mozilla.org/en-US/kb/connection-settings-firefox)'
+---
+
+This guide will show you how to create your own HTTP proxy using Squid, a highly customizable proxy/cache application. An HTTP proxy acts as an intermediary between you and the internet. While connected to your Squid HTTP proxy, you will be able to:
```

Comment:
Can we bake the distro into the opening paragraphs? Like "This guide will show you how to create your own HTTP proxy using Squid on a CentOS 8 Linode. Squid is a highly..."